Library/Formula/fastd.rb

```diff
@@ -0,0 +1,31 @@
+class Fastd < Formula
+  desc "Fast and Secure Tunnelling Daemon"
+  homepage "https://projects.universe-factory.net/projects/fastd"
+  url "https://projects.universe-factory.net/attachments/download/81/fastd-17.tar.xz"
```

Comment:
More problematic than the homepage test failing, you also can't fetch this download link because `curl` performs SSL/TLS validation. I'm afraid we can't include a formula we can't CI fetch, and users won't be able to download locally either.

You'll need to find an official or Linux mirror somewhere that has a valid certificate or is at least reachable for this to be included.